docs/samples/browser-plugin-meetings/app.js

```diff
@@ -701,12 +701,11 @@ const htmlMediaElements = [
 
 
 function cleanUpMedia() {
-  // the local tracks get stopped when we get the unpublished event for them, so here we only need to lose the reference to them
-  localMedia.microphoneTrack = undefined;
-  localMedia.cameraTrack = undefined;
+  localMedia.microphoneStream.stop();
```

Comment:
@mkesavan13, why we are calling .stop explicitly here but not earlier, wanted to understand.

Reply:
The call to `stop` used to be in the handler for the `published-state-update` event, which was removed in the PR. But I've discussed with @mkesavan13 and he will add the event back in a future PR, at which point, we should move the call to `stop` back into the handler and revert this function.